zio-http/src/main/scala/zhttp/service/Client.scala

```diff
@@ -175,12 +174,26 @@ object Client {
     val url: URL       = endpoint._2
   }
 
-  final case class ClientResponse(status: Status, headers: List[Header], private val buffer: ByteBuf)
+  sealed trait Content {
+    type T
+    def getContent: T
+  }
```

Comment:
It's not really clear to me why we need Path dependant types here.

Reply:
indeed, we do not need any of this part.